### PR TITLE
Add iOS support

### PR DESCRIPTION
**Pull request by aspenluxxxy**
Uses `uiopen` on iOS.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -65,6 +68,42 @@ pub use macos::{that, with};
...
+    target_os = "ios",
+    target_os = "linux",
+    target_os = "macos",
+    target_os = "netbsd",
+    target_os = "openbsd",
+    target_os = "solaris",
+    target_os = "windows",
+)))]
+pub fn that<T: AsRef<OsStr> + Sized>(path: T) -> Result<ExitStatus> {
+    panic!(
```
Comment (Byron) [inline src/lib.rs:83]: Could you turn these panics into [compile_error!](https://doc.rust-lang.org/std/macro.compile_error.html)?
I don’t think anyone wants to learn about the lack of support at runtime.

Then the function declaration won’t be needed either, the attribute applies directly to the macro (I think).

**Review by Byron — CHANGES_REQUESTED** [review]
Thanks a lot - it’s great to see Rust going to phones now :).

With the tiny fix I requested I will merge and release right away.

File: src/lib.rs (hunk 1 above)
Comment (aspenluxxxy) [reply src/lib.rs]: Done! Might wanna squash-merge this, though.

**Comment by Byron** [comment]
Thanks a lot, I will release everything in v1.6 in a bit.

**Pull request merged**